platforms/ide/tooling-api/src/testFixtures/groovy/org/gradle/integtests/tooling/fixture/ToolingApiSpecification.groovy

```diff
@@ -534,11 +534,8 @@ abstract class ToolingApiSpecification extends Specification implements KotlinDs
     }
 
     private String normalizeDeprecationWarning(String message) {
-        def nextMajorVersion = Integer.parseInt(targetDist.version.version.split("\\.")[0]) + 1
-
         def normalizedLink = DocumentationUtils.normalizeDocumentationLink(message, targetDist.version)
-        def normalizedVersion = normalizedLink.replaceAll("Gradle X", "Gradle ${nextMajorVersion}.0")
```

Comment:
🤔 Should we keep this, maybe it comes handy in the future?

Reply:
IDK, why? Current usages break the experimental branch. There are no other usages. The code can be revived if needed.

Reply:
I don't jave strong opinion, but I see this could be useful in future.
@jvandort would you prefer we keep this logic?

Reply:
We can remove this code. The logic will live on in git history if we want it back. 

These deprecation warnings are printed as a result of calling things like `willBeRemovedInGradle9` in the source code, so its not like the messages will magically change to print out `10` when we change Gradle to version 10